### PR TITLE
Added the OdataBind property

### LINKED ISSUE #381: Adding TeamsTab fails as an invalid request
Please provide the following (and please check them off the list with [x]) before submitting this issue:
- [x ] Expected behavior. Please provide **links to the specific [Microsoft Graph documentation](https://developer.microsoft.com/en-us/graph/docs/concepts/overview)** you used to determine the expected behavior. 
- [x ] Actual behavior. Provide error codes, stack information, and a [Fiddler](http://www.telerik.com/fiddler) capture of the request and response (please remove personally identifiable information before posting).
- [x ] Steps to reproduce the behavior. Include your code, IDE versions, client library versions, and any other information that might be helpful to understand your scenario.

### Expected behavior

According to https://docs.microsoft.com/en-us/graph/api/teamstab-add?view=graph-rest-1.0 and https://developer.microsoft.com/en-us/kaizala/blogs/30daysmsgraph-day-27-use-case-create-a-team/ , the call to add a tab using this library would be

    var apps = await graphServiceClient.Teams[_teamId_].InstalledApps.Request().Expand("TeamsApp").GetAsync();
    var WebApp = apps.Where(x => x.TeamsApp.DisplayName == "Website").Select(y => y.TeamsApp).FirstOrDefault();

    WebApp .ExternalId = "https://graph.microsoft.com/v1.0/appCatalogs/teamsApps/com.microsoft.teamspace.tab.web";

    var newTab = new Microsoft.Graph.TeamsTab()
    {
        DisplayName = "Map",
        TeamsApp = WebApp ,
        Configuration = new Microsoft.Graph.TeamsTabConfiguration()
        {
            EntityId = null,
            ContentUrl = "https://www.bing.com/maps/embed?h=800&w=800&cp=47.640016~-122.13088799999998&lvl=16&typ=s&sty=r&src=SHELL&FORM=MBEDV8",
            WebsiteUrl = "https://binged.it/2xjBS1R",
            RemoveUrl = null,
        }
    };

    var addedTab = await graphServiceClient.Teams[_teamId_].Channels[_channelId_].Tabs.Request().AddAsync(newTab);

### Actual behavior

This returns an ‘Invalid Request’ exception – ‘The navigation bind for the teamsApp was missing in request.’

This can be rectified by implementing a TeamsTab as in Contoso-Airlines-Teams code example from https://github.com/microsoftgraph/contoso-airlines-teams-sample i.e. string TeamsApp, not TeamsApp TeamsApp as in the current implementation.

### Steps to reproduce the behavior

I'm using Microsoft.Graph v1.13.0 and Microsoft.Graph.Core v1.13.0 installed from NuGet.
Use the code snippet above where 

    graphServiceClient = new GraphServiceClient(new DelegateAuthenticationProvider((requestMessage) => {
        requestMessage.Headers.Authorization = new AuthenticationHeaderValue("bearer", token);

        return Task.FromResult(0);
    }));

_teamsId_ is the identifier for the Team, _channelId_ is the channel identifier.

### PR DESCRIPTION
**Pull request by FiromsaA**
Closes #381

### CODE REVIEW

File: src/Microsoft.Graph/Models/Extensions/TeamsTabExtension.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace Microsoft.Graph
+{
+    public partial class TeamsTab 
+    {
+        /// <summary>
+        /// Gets or sets an assiociated existing app with a teams tab.
+        /// </summary>
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName = "teamsApp@odata.bind", Required = Newtonsoft.Json.Required.Default)]
+        public string OdataBind { get; set; }
```
Comment (darrelmiller) [inline src/Microsoft.Graph/Models/Extensions/TeamsTabExtension.cs]:
```suggestion
        public string ODataBind { get; set; }
```

**Comment by MIchaelMainer** [comment]
@FiromsaA Please add a link to the fixed issues by putting `Fixes #<issue number>` so that a link is made between this pull request and the issue.

File: src/Microsoft.Graph/Models/Extensions/TeamsTabExtension.cs (hunk 1 above)
Comment (FiromsaA) [reply src/Microsoft.Graph/Models/Extensions/TeamsTabExtension.cs]: Capitalized ODataBind
Fixes #<508>

**Comment by FiromsaA** [comment]
Fixes #381

**Pull request merged**